java/sage/media/format/FormatParser.java

```diff
@@ -187,6 +189,29 @@ public static ContainerFormat getFileFormat(java.io.File f)
         }
       }
 
+      FormatParserPlugin plugin = FormatParser.getFormatParserPluginInstance();
+            
+      if(plugin != null)
+      {
+        if (sage.Sage.DBG) System.out.println("Using the format detector plugin");
+        
+        try
+        {
+          ContainerFormat pluingFormat = plugin.parseFormat(f);
+            
+          if(pluingFormat.streamFormats != null && pluingFormat.streamFormats.length > 0)
+          {
+            addAdditionalMetadata(f, pluingFormat);
+            return pluingFormat;
+          }
+        }
+        catch(Exception ex)
```

Comment:
Space after catch, and catch a Throwable instead of Exception so nothing gets through.